frontend/ui/widget/touchmenu.lua

```diff
@@ -1027,8 +1027,10 @@ function TouchMenu:search(search_for)
         for i, v in ipairs(item_table) do
             if type(v) == "table" and not v.ignored_by_menu_search then
                 local entry_text = v.text_func and v.text_func() or v.text
-                local indent = text and ((" "):rep(math.min(depth-1, 6)) .. "→ ") or "→ " -- all spaces here are Hair Space U+200A
-                local walk_text = text and (text .. "\n" .. indent .. entry_text) or (indent .. entry_text)
+                is_disabled = is_disabled or v.enabled == false or (v.enabled_func and v.enabled_func() == false)
+                local entry_displayed_text = is_disabled and "▒ " .. entry_text or entry_text
```

Comment:
I preferred when you used `"\u{2592} " ... - Medium Shade + Hair Space`.
That's been our usage recently (may be not here with the arrows, but recently in the skimwidget rtl arrows).

Reply:
On average I don't really like the look of `\u{2592}` and such and I prefer to avoid it when reasonable, but the comment saying there's a hair space is quite important either way to avoid future mishaps.

Reply:
Answer please:
(1) ▒  or `\u{2592}`
(2) → or `\u{2192}`
(3) "Hair Space" comment in both lines or in the second line only ("here and above")
https://github.com/koreader/koreader/blob/c433c1bb439535a21cdbad1e60669f42ea782209/frontend/ui/widget/touchmenu.lua#L1031-L1032

Reply:
> (1) ▒  or \u{2592}

Probably the latter; the former looks a bit like a rendering bug. :-)

> (2) → or \u{2192}

I much prefer →
But there's certainly no need to go changing any if they happen to be different, and if @poire-z prefers the latter I'm okay with that too. It might be worth putting the → in the comment alongside the name in that case.

> (3) "Hair Space" comment in both lines or in the second line only ("here and above")

I'd just stick it on each line for the future.